packages/jest-types/src/Config.ts

```diff
@@ -105,6 +105,13 @@ export type DefaultOptions = {
   watchman: boolean;
 };
 
+export type DisplayName =
+  | string
+  | {
+      name?: string;
```

Comment:
should these be optional? Why not required?

Reply:
Oh right. I was confused about it for a second since typescript doesn't do runtime checks and was thinking this was a more accurate depiction of the type that we could receive. Let me amend it.

Reply:
IMO we should normalize this in `normalize` in `jest-config`, so that what's passed into jest might be `displayName: 'www'`, but what comes out and passed to reporters are `displayName: { name: 'www', color: 'white' }`. That's a breaking change though (for custom reporters). But the validation you added to `utils` should be added to normalize so that you're passed either a string or a valid object in a reporter

Reply:
Oh that sounds a good idea. So moving forward just to be clear, in `normalize`, I just validate the value of displayName and if it's an invalid shape, I throw a `createConfigError`? That way, in the reporters, no validation logic is needed then.

Reply:
yup!